README.rst

```diff
@@ -258,6 +258,41 @@ To do so set ``uniform=True`` when calling ``extract``, it's false by default fo
 
 NB rdfa structure is not uniformed yet
 
+Returning HTML node
++++++++++++++++++++
+
+It is also possible to get references to HTML node for every extracted metadata item.
```

Comment:
You should specify that this option is available only for microdata. If a user selects return_html_node = True and syntaxes = [opengraph] it won't see the nodes even though it will expect to